test/scripts/cases/_olla.sh

```
@@ -0,0 +1,311 @@
+        echo "olla-${git_hash}"
+    fi
+    
+    cd "$current_dir"
+}
+
+# Function to build Olla
+build_olla() {
+    print_color "$YELLOW" "Building Olla from source..."
+    
+    # Save current directory
+    local current_dir=$(pwd)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)